changefeedtrigger/blobEventsSampleGenerator.py

```diff
@@ -0,0 +1,48 @@
+from azure.storage.blob import BlobServiceClient
+import random
+import time
+from .settings import *
+
+# The purpose of this file is to randomly generate changefeed events, either BlobCreated or BlobDeleted
+# This file is not required to run the changefeedSample.py
+
+
+def main():
+    container_name = "test-changefeed-container"
+    blobs = ["blob1", "blob2", "blob3"]
+    message = "Lorem ipsum"
+
+    blob_service_client = BlobServiceClient.from_connection_string(CONNECTION_STRING)
+
+    try:
+        container_client = blob_service_client.create_container(container_name)
+    except:
```

Comment:
should this catch specific exception ? (as well in other places like that)

Reply:
yeah I think I want it to be a ResourceNotFoundError

Reply:
This shouldn't be ResourceNotFoundError, because this is when you create a client, and the client might exist. If I remember properly the name is ResourceExistsError

---
In reply to: [470848831](https://github.com/wastore/azure-storage-samples-for-python/pull/7#discussion_r470848831) [](ancestors = 470848831)